tox.ini

```diff
@@ -24,7 +25,8 @@ allowlist_externals =
     timeout
     grep
 commands =
-    coverage run -m unittest discover []
+    python3 -m pip install .
```

Comment:
Why is this needed? (probably a stupid question...)

Reply:
there's no stupid questions :)

tox would run `python setup.py install` for building a sdist, and since we'd like to build via pyproject.toml, `skipsdist = True` was added, and the dependencies installed with pip install.

see https://stackoverflow.com/questions/18962403/how-do-i-run-tox-in-a-project-that-has-no-setup-py